Client/Frontend/Home/FirefoxHomeViewController.swift

```diff
@@ -86,9 +86,8 @@ class FirefoxHomeViewController: UICollectionViewController, HomePanel {
         collectionView?.backgroundColor = .clear
         view.addSubview(wallpaperView)
 
-        if shouldShowDefaultBrowserCard {
-            showDefaultBrowserCard()
-        }
+        /// As long as there's a message, always show it.
+        showDefaultBrowserCard()
```

Comment:
This is pending PM review of behaviour and the default behaviour we were exploring. 

also, if we're doing the heavy lifting at the start, we could have a function called `shouldShow(surface: .newTabCard)` or something like that in the manager.

Reply:
After speaking with Product, we've decided that we should show the original behavior the banner had instead of not showing the surface at all when there are no messages. 

Already done.